### DISCUSSION TITLE
视频截图功能希望能增加一个复制到剪贴板的选项

### DISCUSSION THREAD

**Discussion by SekiBetu** [the1812/Bilibili-Evolved#2409] [功能建议 / Ideas]
每次要保存到本地之后再从比如QQ之类的软件发出去太慢了，截图可以复制到剪贴板的话会很方便，或者从新标签页打开截图之类的

**Comment by the1812** [comment]
图上右键不行吗

**Reply by SekiBetu** [reply]
![image](https://user-images.githubusercontent.com/38283893/138065194-f2b19e51-70aa-4d5a-8cae-272132872aec.png)
v2这遮罩严严实实的，在哪右键都不行

**Reply by the1812** [reply]
啊 可能我忘了设鼠标穿透了

**Reply by SekiBetu** [reply]
any updates?

**Reply by the1812** [reply]
已更新

**Reply by SekiBetu** [reply]
好像没用，我问一下，我之前是v2 branch装的，然后里面的在线找的功能一个一个加进去的，然后导出配置，删除脚本，安装preview branch的，然后导入配置，这时候，配置里面是不是还是v2 branch的配置更新地址，所以导致无法更新到这个修复后的版本

**Reply by SekiBetu** [reply]
[settings.zip](https://github.com/the1812/Bilibili-Evolved/files/7469209/settings.zip)
这是我的配置文件

**Reply by SekiBetu** [reply]
![image](https://user-images.githubusercontent.com/38283893/140102552-f04e78f5-5342-4354-88af-1d0e3e1563e9.png)
看到了，原来在最下面，那我替换下就没事了，问题不大

**Reply by SekiBetu** [reply]
为什么不用 
`https://raw.githubusercontent.com/the1812/Bilibili-Evolved/preview/registry/dist/components/video/player/screenshot.js` 
而是用
`https://github.com/the1812/Bilibili-Evolved/raw/preview/registry/dist/components/video/player/screenshot.js` 
啊，这样还多了一层跳转

**Reply by SekiBetu** [reply]
我还看到这些插件中的 `alwaysUpdate` 值是 `false` ，是默认不更新的吗

**Reply by the1812** [reply]
`alwaysUpdate` 为 `true` 时, 无视 `lastUpdateCheck`, 总是会检查更新

raw 的链接确实得换下
